### PR TITLE
Migrate templates

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: iogt/templates/core/section_page.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 {% extends "base.html" %}
 
-{% load static core_tags %}
+{% load static core_tags i18n %}
```
Comment (hodgestar) [inline iogt/templates/core/section_page.html]: I'm wondering why we're loading `i18n` here since I don't see any `trans` or `blocktrans` being used?

**Comment by hodgestar** [comment]
The code changes look good to me. Could you perhaps attach images of a suitable page from the old UC site and this site? I'd help a lot for getting a sense of what the new layouts look like.

I'd also like @miltontony to look at this briefly (just in case I'm missing anything).

**Comment by miltontony** [comment]
:+1:

**Comment by rudigiesler** [comment]
@hodgestar Keep in mind that there's no CSS.. 

Main page (old):
screencapture-00-iogt-apollo-unicore-io-1461235088982

Main page (new):
screencapture-localhost-8000-1461235063653

Section page (old):
screencapture-00-iogt-apollo-unicore-io-section-emergency-1461235199923

Section page (new):
screencapture-localhost-8000-english-emergency-information-1461235205859

Article page (old):
screencapture-00-iogt-apollo-unicore-io-page-571f1bb6a72b4ff79687359399d01838-1461235256236

Article page (new):
screencapture-localhost-8000-english-emergency-information-what-you-need-know-about-zika-virus-where-zika-virus-1461235272752

**Comment by hodgestar** [comment]
I think we need to get the CSS back and some of the images need to be displayed, but I'm still :+1: on landing this and iterating in future PRs.

@peterp Would you like to be involved in sorting out the CSS?

**Comment by peterp** [comment]
@hodgestar Yeah! Could I use this to start the conversation about SMACSS and BEM?

**Comment by hodgestar** [comment]
@peterp Sure, although I'd prefer it if we picked a direction fairly soon (we could discuss pro's and con's forever -- happy to follow your and @justinvdm's judgement).

**Pull request merged**